project/AppDependencies.scala

```diff
@@ -6,8 +6,8 @@ object AppDependencies {
   val silencerVersion = "1.7.0"
 
   val compile = Seq(
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28" % "5.3.0",
-    "uk.gov.hmrc"       %% "play-frontend-hmrc"         % "0.79.0-play-28",
+    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28" % "5.7.0",
```

Comment:
how did you test this?

Reply:
I ran SBT tests and ran thru some of the MRN journey. Was there a problem...?